requirements.txt

```diff
@@ -1,2 +1,3 @@
 PyYAML==3.11
+yapf==0.10.0
```

Comment:
As discussed thought this applied to setup.py but only applies to travis configuration.